src/content/app/genome-browser/hooks/useGenomeBrowserTracks.ts

```diff
@@ -66,7 +71,9 @@ const useGenomeBrowserTracks = () => {
   useEffect(() => {
     // skip the effect if track categories have not yet been fetched
     // or if they have already been set
-    if (!trackCategories || !focusObject || trackSettingsForGenome) {
+    if (
+      !(trackCategories && !focusObject && trackSettingsForGenome && genomeId)
```

Comment:
![image](https://user-images.githubusercontent.com/6834224/191506825-0bbb5c33-57db-4ee4-94a1-fa4a7fc5bb8d.png)

Reply:
I'm sorry; but the change is just not the same:

![image](https://user-images.githubusercontent.com/6834224/191509973-cbceea34-29fa-4ba0-9818-09a920584b72.png)

The original comment says:

```
// skip the effect if track categories have not yet been fetched
// or if they have already been set
```
And the original condition was saying to return if `trackSettingsForGenome` were truthy, not falsey

Reply:
oops! Sorry, I think I should have avoided this change. I've reverted it and just included a check for `genomeId`